CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
-### ZeroNet 0.7.2 (2020-09-?) Rev4206?
-
+### zeronet-conservancy 0.7.3 (2022-01-07) Rev5000
+- forked from the latest py3 branch of ZeroNet
+- onion v3 support (thanks to @anonymoose, @zeroseed and @geekless)
+- partial readme rewrite (thanks to @mitya57)
 
+### ZeroNet 0.7.2 (2020-09-?) Rev4206?
```

Comment:
I doubt it was the same revision as 0.7.1.

Reply:
yeah , but i didn't invent that part of changelog . i haven't been around ZN development at the time . not sure if i can invest time into fixing the old changelog right now , but i'll try to get back to it eventually

Reply:
fixed it anyway